shoop/admin/modules/orders/static_src/create/reducers/customer.js

```diff
@@ -31,7 +31,13 @@ function setAddressProperty(state, {payload}) {
 }
 
 export default handleActions({
-    setCustomer: ((state, {payload}) => payload),
+    setCustomer: ((state, {payload}) => _.assign(state, {
```

Comment:
If I do that saveAddress and shipToBillingAddress is set to it's default value since those are missing from payload.